general_itests/deployments_json.feature

```diff
@@ -5,3 +5,8 @@ Feature: Per-Service Deployments.json can be written and read back
 	When paasta stop is run against the repo
 	 And we generate deployments.json for that service
         Then that deployments.json can be read back correctly
+
+    Scenario: New services have a desired_state of "start"
+	Given a test git repo is setup with commits
```

Comment:
My understanding is that the context is cleared between the scenarios (and the tests appear to confirm that). This is also why I removed the `shutil.rmtree` call in the corresponding step (it would cause issues if you try and chain multiple 'then' steps).